pipeline/buildpacks/0.1/README.md

```diff
@@ -0,0 +1,89 @@
+# Buildpacks
+
+This pipeline builds source into a container image using [Cloud Native Buildpacks](https://buildpacks.io). To do that, it uses [builders](https://buildpacks.io/docs/concepts/components/builder/#what-is-a-builder) to run buildpacks against your application source.
+
+> _**What are Cloud Native Buildpacks?**_
+> 
+> _Cloud Native Buildpacks are pluggable, modular tools that transform application source code into OCI images. They replace Dockerfiles in the app development lifecycle, and enable for swift rebasing of images and modular control over images (through the use of builders), among other benefits._
+
+## Dependencies
+
+- [`git-clone` task](https://github.com/tektoncd/catalog/tree/master/task/git-clone) 0.2 or newer
```

Comment:
I guess you need to pin the version of `git-clone` because `pipelines.minVersion` for `git-clone` `0.3` is `0.21.0` and `pipelines.minVersion` for `buildpacks` pipeline is `0.17.0` which can create compatibility issues

Reply:
Good call. Will update this and in the future will likely bump up to minVersion `0.21.0` to support the latest versions of tasks.

Reply:
Sounds good :+1: